pytest_inmanta/plugin.py

```diff
@@ -336,15 +341,22 @@ def deploy_resource(self, resource_type: str, status=const.ResourceState.deploye
 
         assert ctx.status == status
         self.finalize_context(ctx)
+        if return_logs:
+            return res, ctx.logs
```

Comment:
I would prefer not to have variable number of return arguments. 

It would be cleaner to keep the context in a variable and have a get_last_logs and get_last_context 

@bart wdyt?

Reply:
Alternative would be to have another method name for the one that exposes its context.